recipes/wn/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "wn" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wn-{{ version }}.tar.gz
+  sha256: aeaaf3194885d57419cc4eb189cc09ec226c23b4af5a5153e270c2ae1b505168
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - flit-core >=3.4,<4
+    - pip
+  run:
+    - python >=3.7
+    - requests
+    - tomli
+    - importlib_resources
```

Comment:
According to the upstream `pyproject.toml`, this is only needed for `python_version < '3.7'`, which you're not building here?
```suggestion
```

Reply:
@dopplershift Thank you for point it out. I have added your suggested change.